libs/designer-ui/src/lib/card/subgraphCard/subgraphCard.less

```diff
@@ -32,11 +32,38 @@
     color: white;
     flex-grow: 1;
     align-self: center;
-    display: inline-block;
-    font: 14px/20px normal @semibold-font-family;
```

Comment:
It looks like you removed the `font` prop, was it causing issues?
From your screenshot the large sized node looks fine without it but the small sized node is incorrect

Reply:
oh, good catch! I removed it there and added it to a new class because it was making the error message appear bold and I didnt notice it affected the small node, let me fix that